examples/dt/uni01alpha/README.md

```diff
@@ -0,0 +1,259 @@
+# Deployed Topology - Alpha
+
+This document contains a list of integration test suites that would be
+executed against the below specified topology of Red Hat OpenStack Services
+on OpenShift. It also contains a collection of custom resources (CRs) for
+deploying the test environment.
+
+## Purpose
+
+This topology is used for executing integration tests that evaluate the
+`default` backends of the below mentioned services.
+
+## Environment
+
+### Nodes
+
+| Role | Machine Type | Count |
+| ---- | ------------ | ----- |
+| Compact OpenShift | vm | 3 |
+| OpenStack Compute | vm | 2 |
+| Networker         | vm | 3 |
+| Test nodes        | vm | 2 |
+
+### Networks
+
+| Name | Type | Interface |
+| ---- | ---- | --------- |
+| Provisioning | untagged | nic1 |
+| Machine | untagged | nic2 |
+| RH OSP | trunk | nic3 |
+
+#### Networks in RH OSP
+
+| Name | Type |
+| ---- | ---- |
+| ctlplane | untagged |
+| internalapi | VLAN tagged |
+| Storage | VLAN tagged |
+| Tenant | VLAN tagged |
+
+### Services, enabled features and configurations
+
+| Service          | configuration    | Lock-in coverage?  |
+| ---------------- | ---------------- | ------------------ |
+| Cinder           | LVM/iSCSI/lioadm | Must have          |
+| Cinder Backup    | Swift            | Must have          |
+| Glance           | Swift            | Must have          |
+| Swift            | (default)        | Must have          |
+| Octavia          | (amphora)        | Must have          |
+| Horizon          | N/A              | Must have          |
+| Barbican         | (default)        | Must have          |
+| Telemetry        |                  | Must have          |
+| Ironic           |                  | Must have          |
+| Neutron          | OVN - AZs        | Must have          |
+
+#### Support services
+
+The following table lists services which are not the main focus of the testing
+(which may be covered by additional scenarios), but are required for the DT to
+work properly and can be deployed with any/default configuration.
+
+| Service          | Reason            |
+| ---------------- |------------------ |
+| Nova             | needed by scenario testing |
+| Keystone         | needed by all services     |
+
+### Additional configuration
+
+- Always-on, default services and features: TLSe
+- Two additional fake baremetal nodes
+- Availability zones for OVN (zone-1 & zone-2)
+- Logical volume with the name cinder-volumes exists on a OpenShift node.
+- iSCSI service is enabled on all OpenShift nodes.
+- Multipath service is enabled on all OpenShift nodes.
+
+#### iSCSI
+
+It is assumed *iSCSI* services are enabled in all nodes participating in the
+Red Hat OpenShift cluster. If not, a `MachineConfig` similar to the below one
+is applied. The node needs to be *rebooted* after applying the configuration.
```

Comment:
Applying the MachineConfig will automatically trigger the reboot. There's no need for CI to trigger it. However, something in the ci-framework will need to handle the delay caused by the reboot which happens as a side effect of applying a MachineConfig.

Reply:
Thanks for the additional information. I have corrected the statement to include a reboot. As part of this test run, the OpenShift cluster is configured upfront (during installation) with
- iSCSI
- Multipath
- LVM

This was to done to overcome the limitation we would experience if there is a reboot in between the deployment of OpenStack services.